src/style.css

```
@@ -0,0 +1,63 @@
+h1 {
+  display: none;
+}
+
+* {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+}
+
+.stars--1 :first-child {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--2 :nth-child(1),
+.stars--2 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--3 :nth-child(1),
+.stars--3 :nth-child(2),
+.stars--3 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
```

```suggestion
.stars--1 :first-child(1)
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3) {
  background-image: url(images/star-active.svg);
}
```
для трьох кейсів я записав, завдання для тебе записати для решти

---

зробила, перевір будь ласка, чи все ок